content/le-droit-a-l-image-des-eleves.json

```
@@ -0,0 +1,8 @@
+{
+  "title": "Le droit à l'image des élèves",
+  "description": "Comme tout citoyen, les élèves disposent d'un droit à l'image. Comment le respecter au mieux en tant qu'enseignant ? C'est ce que nous allons voir dans cette vidéos",
```

Petite erreur dans la description ?

```suggestion
  "description": "Comme tout citoyen, les élèves disposent d'un droit à l'image. Comment le respecter au mieux en tant qu'enseignant ? C'est ce que nous allons voir dans cette vidéo.",
```